citation-backoffice/src/logic/objects.js

```diff
@@ -39,14 +40,17 @@ export function writeObject(type, data) {
 			return `${key}: ${formatedData}`;
 		});
 	}
-
+	data[subType].__newId__ = data[subType].__id__;
+	data[subType].__id__ = id;
 	return dispatch => {
 		return mutation(`{edit${type}(${formatData(data)}) {${fields[type]}}}`)
-			.then(response => dispatch(loadObjectSuccess({
-				type,
-				id: response.data.editObject.__id__,
-				data: response.data.editObject
-			})));
+			.then(response => {
+				dispatch(loadObjectSuccess({
```

Comment:
warning, by adding `{}` you break the automatic return and then break the promise chain, was that what you wanted?